### PR TITLE
Add new Azure Security Center templates to logic apps

### PR DESCRIPTION
**Pull request by keren-shani**
Add 2 new templates for logic apps:
1) For ASC Alert:
Trigger: Azure Security Center Alert connector
Action: Send an email

2) For ASC Recommendation:
Trigger: Azure Security Center Assessment connector
Action: Send an email

The connectors I use are preview connectors

### CODE REVIEW

File: templates/asc-alert-send-notification-email.json [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    "id": "/providers/Microsoft.Logic/galleries/public/templates/d27feb15-dc1d-4068-93c5-d9dca32806cd",
+    "type": "Microsoft.Logic/galleries/templates",
+    "name": "d27feb15-dc1d-4068-93c5-d9dca32806cd",
+    "properties": {
+        "author": {
+            "displayName": "Microsoft"
+        },
+        "categoryNames": [
+            "general",
+			"security"
```
Comment (derek1ee) [inline templates/asc-alert-send-notification-email.json]: Indentation (this and the rest of the files)

File: templates/asc-alert-send-notification-email.json [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        ],
+        "description": "Send an email notification with the alert details when a threat is detected",
+        "displayName": "Get a notification email when Security Center detects a threat",
```
Comment (derek1ee) [inline templates/asc-alert-send-notification-email.json]: Make sure there's no text clipping in UI

File: templates/asc-alert-send-notification-email.json (hunk 1 above)
Comment (keren-shani) [reply templates/asc-alert-send-notification-email.json]: I fixed the indentation

File: templates/asc-alert-send-notification-email.json (hunk 2 above)
Comment (keren-shani) [reply templates/asc-alert-send-notification-email.json]: I checked that the texts are not cut in the UI if that is what you mean.
Is there a specific way to check that there is no text clipping in UI?

File: templates/asc-alert-send-notification-email.json [hunk 3]
```diff
@@ -0,0 +1,99 @@
+{
```
Comment (keren-shani) [inline templates/asc-alert-send-notification-email.json]: Is there a way to define my templates as "Preview"?
The connectors I use are "Preview" connectors

Comment (derek1ee) [reply templates/asc-alert-send-notification-email.json]: Unfortunately no

File: templates/manifest.json [hunk 4]
```diff
@@ -1,5 +1,7 @@
 [
     "as2-response.json",
+    "asc-alert-send-notification-email.json",
+    "asc-recommendation-send-notification-email.json",
```
Comment (derek1ee) [inline templates/manifest.json]: nit: order alphabetically

**Pull request merged**